inc/template-tags.php

```diff
@@ -258,7 +258,11 @@ function siteorigin_unwind_post_meta() {
 	} ?>
 
 	<?php if ( siteorigin_setting( 'blog_display_date' ) ) { ?>
-		<span class="entry-date"><?php the_time( 'M d, Y' ); ?></span>
+		<span class="entry-date">
+			<?php echo ( ! is_singular() ) ? '<a href="' . get_the_permalink() . '">' : ''; ?>
```

Comment:
To be a little more SEO friendly, adding in `title=' . the_title_attribute() . '` here would be ideal.